src/content/docs/en/linux/ssh.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: SSH Secure Shell
+description: "An overview of the SSH utility."
+---
+
+import Alert from "../../../../components/Docs/Alert.astro";
+
+Secure Shell ("SSH") is one of the most common utilities when access to a remote device is needed. It's encrypted, powerful, and simple to understand.
+
+## SSH overview
+SSH uses a client/server model where one computer acts as an SSH server[^1] and allows other computers to connect.
+
+![Example SSH Client and Server](/assets/ssh.png)
+
+## Connecting to an SSH Server
+To access a device remotely, the `ssh` command is run from [the shell](/en/linux/shell), mostly used with this format:
+
+```zsh
+ssh [username@]<server url>
+```
+If you don't specify a username then SSH will attempt to connect you using the username with which you are logged in. Here are some examples:
+- `ssh myuser@myserver.com`
+    - This will attempt to connect to the server at myserver.com using the username myuser
+- `ssh myuser@192.168.1.2`
+    - This is just like the example above except you can also specify an IP address instead of a hostname
+- `ssh myserver.com`
+    - This will attempt to connect to the SSH server at myserver.com using your current username
+
+When using any of the above commands, once connected to the remote server you'll be prompted for a password.
+
+## Running Your Own SSH Server
+Most modern Linux distros come with the required software to host your own SSH server. To allow users to connect to your machine, start the SSH server application using
+```zsh
+sudo systemctl start sshd
+```
+Note that the above command uses `sudo`. The `sudo` command will execute the following commands as root (the superuser). For many commands that involve starting and stopping servers, the permission of the system's superuser is necessary. You can read more about the `sudo` command [here](en/linux/permissions). The above command also uses the `systemctl` command. This command is mostly used to manage servers or other Linux services.

```

Comment:
maybe use systemctl without sudo? this will invoke polkit which may be easier for users